drivers/pci/controller/vmd.c

```diff
@@ -31,9 +31,34 @@
 #define PCI_REG_VMLOCK		0x70
 #define MB2_SHADOW_EN(vmlock)	(vmlock & 0x2)
 
+#define PCI_VMD_PRIMARY_PCH_BUS 0x80
+#define PCI_REG_BUSRANGE0 0xC8
+#define PCI_REG_BUSRANGE1 0xCC
+#define PCI_MEMBAR1_OFFSET 0xD0
+#define PCI_MEMBAR2_OFFSET1 0xD8
+#define PCI_MEMBAR2_OFFSET2 0xDC
+#define VMD_ROOTBUS_RANGE_END(busr) ((busr >> 8) & 0xff)
+#define VMD_ROOTBUS_RANGE_START(busr) (busr & 0x00ff)
+
 #define MB2_SHADOW_OFFSET	0x2000
 #define MB2_SHADOW_SIZE		16
 
+enum vmd_resource {
+	VMD_RESOURCE_CFGBAR,
```

Comment:
`VMD_RESOURCE_CFGBAR = 0` to force initialization from 0

```suggestion
VMD_RESOURCE_CFGBAR = 0,
```

Reply:
Fixed.